share/revng/pipelines/yield-call-graph.yml

```
@@ -0,0 +1,15 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+Containers:
+  - Name: cross-relations.yml
+    Type: BinaryCrossRelations
+    Role: CrossRelations
+Branches:
+  - From: Isolate
+    Steps:
+      - Name: ProcessCallGraph
+        Pipes:
+          - Type: process-call-graph
```

Pipe names are `CamelCase`
```suggestion
          - Type: ProcessCallGraph
```

---

Done: https://github.com/revng/revng/pull/197/commits/ea09d706e8e2291973e69882ba0704d94d33796d.